temba/flows/models.py

```diff
@@ -1759,7 +1769,7 @@ def start_msg_flow_batch(self, batch_contact_ids, broadcasts, started_flows, sta
                 # create the sms messages
                 created_on = timezone.now()
                 broadcast.send(expressions_context=expressions_context_base, trigger_send=False,
-                               response_to=start_msg, status=INITIALIZING, msg_type=FLOW,
+                               response_to=start_msg, status=INITIALIZING, msg_type=FLOW, metadata=broadcast.metadata,
```

Comment:
as above

Reply:
I think that here we could pass `metadata` because internally it's being handled (https://github.com/Ilhasoft/rapidpro/blob/133993c1b482a53e70d9358d69d094f207042322/temba/msgs/models.py#L471)

Reply:
Why is metadata needed here - aren't the the broadcasts are being created with the quick_replies at https://github.com/rapidpro/rapidpro/pull/600/files/133993c1b482a53e70d9358d69d094f207042322#diff-ac952ec358ed4e019095b25fcac469adR1668 ?